doc/source/user/debugging.rst

```diff
@@ -0,0 +1,32 @@
+==============================
+Debugging
+==============================
+
+
+There is several variables that you can use to get extra debugging from the ansible playbooks as shown in https://docs.ansible.com/ansible/latest/reference_appendices/config.html
+
+Some examples:
+
+.. code-block:: console
+
+    ANSIBLE_VERBOSITY = 3
```

Comment:
I think you should remove the spaces, else it might be a problem.

You should also add `export` at the beginning in each line.